### ISSUE TITLE
Probe failed on CentOS7

### ISSUE THREAD

**Issue by eserte** [PerlAlien/Alien-pkgconf#2]
While running make on a CentOS7 system:
```
cp lib/Alien/pkgconf.pm blib/lib/Alien/pkgconf.pm
"/opt/perl-5.24.1/bin/perl5.24.1" script/fetch.pl
fetch http://distfiles.dereferenced.org/pkgconf
fetch http://distfiles.dereferenced.org/pkgconf/pkgconf-1.2.0.tar.xz
write _alien/tar/pkgconf-1.2.0.tar.xz
"/opt/perl-5.24.1/bin/perl5.24.1" script/extract.pl
untar _alien/tar/pkgconf-1.2.0.tar
write _alien/src/pkgconf-1.2.0
"/opt/perl-5.24.1/bin/perl5.24.1" script/probe.pl site /opt/perl-5.24.1/lib/5.24.1/x86_64-linux /opt/perl-5.24.1/lib/site_perl/5.24.1/x86_64-linux 
do not know enough about this OS to probe for correct paths.  Please open a ticket https://github.com/plicease/Alien-pkgconf/issues at script/probe.pl line 262.
make: *** [_alien/01probe.json] Fehler 2
```

**Comment by plicease** [comment]
Thank you for reporting this, it would have been a while before I saw this.  This should now be fixed in 0.08.

**Comment by eserte** [comment]
I can confirm this: 0.08 works fine (tested with perl 5.24.0 and 5.24.1 on a CentOS7 system).
